pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2681,3 +2688,13 @@ func resetCachedListenerConfig(mesh *meshconfig.MeshConfig) {
 func listenerKey(bind string, port int) string {
 	return bind + ":" + strconv.Itoa(port)
 }
+
+func dropAlpnFromList(alpnProtocols []string, alpnToDrop string) []string {
+	newAlpnProtocols := []string{}
```

Comment:
Both are not too different, but this is stylistically preferred from what I know.
```suggestion
	var newAlpnProtocols []string
```